extensions/amp-embedly-card/validator-amp-embedly-card.protoascii

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # embedly-card
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-embedly-card"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {  # <amp-embedly-card>
+  html_format: AMP
+  tag_name: "AMP-EMBEDLY-CARD"
+  satisfies: "amp-embedly-card"
+  requires_extension: "amp-embedly-card"
+  attrs: {
+    name: "data-url"
+    mandatory: true
```

Comment:
You may want to consider adding some restrictions on the url. e.g. `https` only. See the [UrlSpec](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L54) for what can be restricted. It can be set as a `value_url` on the tags spec, e.g.

```
value_url: {
  allowed_protocol: "https"
  allow_relative: false
}
````

Reply:
Added!